README.md

```diff
@@ -58,4 +58,7 @@ curl -X 'POST' "http://CNS-MANAGER-ENDPOINT/1.0.0/registercluster?csiDriverSecre
 ## Functionalities currently offered through cns-manager
 
 * **Storage vMotion for CNS volumes**   
-This feature allows migrating volumes from one datastore to another. Read [here](docs/book/features/storage_vmotion.md) for more details about the feature.
+This feature allows migrating volumes from one datastore to another. Read [here](docs/book/features/storage_vmotion.md) for more details about this feature.
+
+* **Orphan volumes detection & deletion**  
+This feature allows detecting/deleting orphan FCDs that are not being used in any of the registered Kubernetes clusters on the vCenter. Read [here](docs/book/features/orphan_volumes.md) for more details about this feature.
```

Comment:
Actually, FCD is an internal implementation detail of CNS. Can we use terms like orphan volumes?